specification/storagesync/resource-manager/Microsoft.StorageSync/StorageSync/back-compatible.tsp

```diff
@@ -38,6 +38,11 @@ using Microsoft.StorageSync;
 @@Azure.ClientGenerator.Core.Legacy.flattenProperty(ServerEndpointUpdateParameters.properties
 );
 
+#suppress "@azure-tools/typespec-azure-core/no-legacy-usage" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+#suppress "@azure-tools/typespec-azure-core/no-legacy-usage" "Back compatibility"
+@@Azure.ClientGenerator.Core.Legacy.flattenProperty(CloudEndpointUpdateParameters.properties
```

Comment:
This is only kept for backward compatibility when converting the spec from Swagger to TypeSpec. For any newly added specs, do not use flatten anymore.